content/03.use-cases-a.md

```diff
@@ -34,16 +25,16 @@ Some EEB labs have even turned their lab notebooks into shareable websites [@url
 GitHub supports a broad set of mechanisms for hosting educational materials.
 The entire process of running a course, workshop, or even just a lecture, can all be done openly on GitHub.
 For example, organizing a course could be broken down into: 1) developing the material (i.e., slides, examples, relevant readings, labs, etc.), 2) hosting the course on some online platform for students to access, 3) delivering the content, 4) accepting student work submissions and 5) returning graded material.
```

Comment:
My idea about [Curriculum development](https://github.com/SORTEE-Github-Hackathon/manuscript/discussions/183), which I previously discussed with @DrMattG  can be mentioned here, if you decide to add it. I just want to remind it. If not please ignore this message.